contentgrid-automations-rest/src/main/java/com/contentgrid/automations/rest/AutomationsRestController.java

```diff
@@ -25,20 +27,27 @@ public class AutomationsRestController {
     private AutomationsModel model;
     @NonNull
     private final AutomationRepresentationModelAssembler assembler;
+    @NonNull
+    private final AbacContextSupplier abacContextSupplier;
+    @NonNull
+    private final AutomationModelVisitor visitor = new AutomationModelVisitor();
```

Comment:
```suggestion
    private static final AutomationModelVisitor visitor = new AutomationModelVisitor();
```

If this is final, and assigned directly: it's a constant.